packages/celotool/src/lib/geth.ts

```diff
@@ -8,7 +8,7 @@ import { unlockAccount } from '@celo/walletkit'
 import BigNumber from 'bignumber.js'
 import { spawn } from 'child_process'
 import fs from 'fs'
-import { range } from 'lodash'
+import _, { range } from 'lodash'
```

Comment:
Only import merge instead of `_`.
```suggestion
import { range, merge } from 'lodash'
```